bin/phpcs-requirelist-commands.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, no-process-exit */
+const spawnSync = require( 'child_process' ).spawnSync;
+const requirelist = require( './phpcs-requirelist' );
+
+console.log( 'PHPCBF... Please expect this to take minutes.' );
+spawnSync( 'vendor/bin/phpcbf', [ ...requirelist ], {
```

Comment:
Throw a `-p` in there to get progress indicators?

Same on line 15?

Reply:
I left them off help reduce any potential scrollback to keep the console output operative.

Cherry-picks trip up since the commit happens without running any of the checks (even though they were ran when the commit was originally done). This is likely something we can fix, but haven't got to the top of the stack yet. https://github.com/Automattic/jetpack/blob/5a3b07954c47081decb1f07102633708469172e5/bin/prepare-commit-msg.js#L11

Reply:
The progress meter shouldn't add too many lines to the scrollback, and it's nice to get a sense that it's progressing and how much longer it could take. But I won't insist.

I've found an interactive rebase (`git rebase -i`) and using "reword" on everything seems to work around the no-verification thing. I don't know if there's an easier way.

Reply:
Added the progress indicators and you're right—they make a much better experience.

Care to review again and approve if you like it, @anomiex?

Reply:
Would you mind terribly fixing the "[not verified]" on the cherry-picked patch? As I mentioned, doing a `git rebase -i` and using "reword" on all the patches seems to let you do that.

Other than that, looks good to me!

Reply:
We can fix it on merge. The commits are squashed, so the actual commit wouldn't have it.

Reply:
Oh, duh. Sorry, I'm still getting used to the "squash merge" workflow.